### DISCUSSION TITLE
How to get the default state of a block

### DISCUSSION THREAD

**Discussion by mikuhl-dev** [Minestom/Minestom#1105] [Help]
I am trying to get the default state of a block to know which properties set on a block are not default.

**Comment by MrlnHi** [comment]
Block.ANVIL.properties()

**Reply by mikuhl-dev** [reply]
But if you have an arbitrary block from the world and can't necessarily hard code it's name?

**Comment by kiip1** [comment]
```java
Block.fromNamespaceId(<block>.namespace()).properties();
```
